trview.ui/TextArea.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+#include "StackPanel.h"
+
+namespace trview
+{
+    namespace ui
+    {
+        class Label;
+
+        /// A text area is a number of lines of text.
+        class TextArea final : public Window
+        {
+        public:
+            /// Create a text area.
+            /// @param position The position to place the control.
+            /// @param size The size of the control.
+            /// @param background_colour The background colour of the text area.
+            /// @param text_colour The text colour for the text area.
+            explicit TextArea(const Point& position, const Size& size, const Colour& background_colour, const Colour& text_colour);
+
+            /// Handle character input from the keyboard.
+            void handle_char(uint16_t character);
```

Comment:
handle_char should just come from the UI and not have to be set up by the owner.

Reply:
Added character handling to the Control class - it is overridden here. Removed the calls to handle_char and the extra keyboard in the RouteWindow class.